src/eal.cpp

```diff
@@ -80,32 +83,62 @@ namespace eal {
       throw invalid_argument("Invalid input coeffs, expected three vectors.");
     }
 
-    vector<double> coordinate = {0.0, 0.0, 0.0};  
-    coordinate[0] = evaluatePolynomial(coeffs[0], time); // X 
+    vector<double> coordinate = {0.0, 0.0, 0.0};
+    coordinate[0] = evaluatePolynomial(coeffs[0], time); // X
     coordinate[1] = evaluatePolynomial(coeffs[1], time); // Y
     coordinate[2] = evaluatePolynomial(coeffs[2], time); // Z
 
     return coordinate;
   }
 
 
-  // Velocity Function 
+  // Velocity Function
   // Takes the coefficients from the position equation
   vector<double> getVelocity(vector<vector<double>> coeffs, double time) {
     vector<double> coordinate = {0.0, 0.0, 0.0};
     return coordinate;
   }
 
-  // Rotation Data Functions
 
   // Rotation Data Functions
-  vector<double> getRotation(string from, string to, vector<vector<double>> rotations,
+  vector<double> getRotation(vector<vector<double>> rotations,
```

Comment:
From what I understand, this should be similar to positional interpolation except for a 4th dimension and extra normalization steps. Could be wrong.

Reply:
I don't have a complete grasp of this either, but it looks like eigen also has a specific method for interpolating between quaternions called `slerp` for spherical linear interpolation: 
http://eigen.tuxfamily.org/dox/classEigen_1_1QuaternionBase.html#ac840bde67d22f2deca330561c65d144e
See: https://en.wikipedia.org/wiki/Slerp

Reply:
Googling around, there are some discussions on stack overflow about how doing a normal linear interpolation can have undesired results, but these are all in the context of using quaternions for graphics, so I'm not sure whether or not this applies to our usecase.

Reply:
Yes, so there are two ways to perform linear interpolation on quaternions: slerp and lerp, which are two different forms of interpolation (and from what I know, this form of straight linear interpolation is lerp, if we want spherical, then slerp will do). The gist of what I want to know: Do we one only one or the other for whatever reason,

Reply:
http://web.mit.edu/2.998/www/QuaternionReport1.pdf 

pg. 40-43 seem to explain the two and the results

Reply:
:+1: On the link to the QuaternionReport resource. I don't know the answer to your question, but would lean toward both?

Reply:
From reading, the main reason to not care about slerp is if our angles have predominately small deltas. In which case, the slower slerp won't really win us anything.

Reply:
Another good write up on differences http://number-none.com/product/Hacking%20Quaternions/.  Quantifies how inaccurate lerp actually is to slerp along with other pros in cons. Keep in mind that this is from a game developers perspective that care way more about real-time efficiency than we do.

Reply:
I would lean towards allowing both methods. For rigorous scientific computations I'd be tempted to use slerp. For quick rough computations lerp may be worth the loss in efficiency.

I think implementing slerp interpolation can be handled in another PR.

Reply:
:+1: